go/libkb/merkle_client.go

```diff
@@ -412,21 +694,69 @@ func (mc *MerkleClient) findValidKIDAndSig(root *MerkleRoot) (keybase1.KID, stri
 	return nilKID, "", MerkleClientError{"no known verifying key"}
 }
 
-func (mc *MerkleClient) verifyRoot(ctx context.Context, root *MerkleRoot, seqnoWhenCalled Seqno) error {
+func (mc *MerkleClient) verifySkipSequence(ctx context.Context, ss SkipSequence, thisRoot *MerkleRoot, lastRoot *MerkleRoot) (err error) {
+	defer mc.G().CTrace(ctx, "MerkleClient#verifySkipSequence", func() error { return err })()
+
+	// In this case, the server did not return a skip sequence. It's OK if
+	// the last known root is too old. It's not OK if the last known root is
+	// from after the server starting providing skip pointers.
+	if ss == nil {
+		mc.G().Log.CDebugf(ctx, "| nil SkipSequence")
+		fss := mc.FirstSeqnoWithSkips()
+		if lastRoot == nil {
+			mc.G().Log.CDebugf(ctx, "| lastRoot==nil, so OK")
+			return nil
+		}
```

Comment:
If we hardcode the first skip seqno in prod, we might want to proceed here even if `lastRoot == nil`. Maybe make this short-circuit conditional on `lastRoot == nil && fss < 0`, so that we don't forget to change it later?